src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4154,15 +4157,20 @@ private ParameterSyntax ParseParameter()
                 {
                     equals = this.EatToken(SyntaxKind.EqualsToken);
                 }
-                exclamation = exclamation is null ? null : CheckFeatureAvailability(exclamation, MessageID.IDS_ParameterNullChecking);
+                if (exclamationExclamation != null)
+                {
+                    exclamationExclamation = (exclamationExclamation.Kind != SyntaxKind.ExclamationExclamationToken)
+                        ? ConvertToMissingWithTrailingTrivia(exclamationExclamation, SyntaxKind.ExclamationExclamationToken)
```

Comment:
>ConvertToMissingWithTrailingTrivia(exclamationExclamation, SyntaxKind.ExclamationExclamationToken) [](start = 26, length = 98)

Should `MergeTokens()` be handling, rather than requiring each caller to do so?

Reply:
Thanks for the tip, I've modified MergeTokens to handle the conversion. Shouldn't have merged so early, it'll be in the next PR.